src/routes/layouts/login/index.js

```diff
@@ -182,15 +182,11 @@ export default class Profile extends Component {
       { isLoggedIn && this.profileDropdown() }
 
       { !isLoggedIn &&
-        <a className={'item text'} href='/pricing'>Plans</a>
-      }
-
-      { !isLoggedIn &&
-        <a className={'item text'} href='/models'>Explore</a>
```

Comment:
Why are you getting rid of the header link to the public models page? Or am I misreading this?